{
  "cveId": "CVE-2024-47035",
  "version": "1.0.0",
  "timestamp": "2024-12-06T11:32:07.789868+00:00",
  "description": "In vring_init of external/headers/include/virtio/virtio_ring.h, there is a possible out of bounds write due to a logic error in the code. This could lead to local escalation of privilege with no additional execution privileges needed. User interaction is not needed for exploitation.",
  "keyphrases": {
    "rootcause": "logic error in the code",
    "weakness": "out of bounds write",
    "impact": "local escalation of privilege",
    "vector": "",
    "attacker": "",
    "product": "external/headers/include/virtio/virtio_ring.h",
    "version": "",
    "component": "vring_init"
  },
  "mitreTechnicalImpacts": []
}